free-courses-en.md

```diff
@@ -235,6 +235,7 @@
 
 * [CS50 2019 - Games Track](https://www.youtube.com/playlist?list=PLhQjrBD2T382mHvZB-hSYWvoLzYQzT_Pb) - David J. Malan (Harvard OpenCourseWare)
 * [Unity Beginner Fundamentals](https://learn.unity.com/course/unity-beginner-fundamentals)(Unity Learn)
+* [Unity Manual](https://docs.unity3d.com/Manual/index.html)(Unity Manual)
```

Comment:
The manual appears to be closer to a book than a course in my opinion.

We prefer to use the title as defined by the resource, please don't create your own title.

Could you please remove the redundant part of the URL at the end?
> always prefer the shortest link: http://example.com/dir/ is better than http://example.com/dir/index.html  
> \- https://github.com/EbookFoundation/free-programming-books/blob/master/CONTRIBUTING.md#guidelines

```suggestion
* [Unity User Manual](https://docs.unity3d.com/Manual/)
```